terraform/prod/key-server/main.tf

```diff
@@ -42,6 +42,8 @@ module "en" {
     }
     exposure = {
       OBSERVABILITY_EXPORTER = "NOOP"
+      MAX_KEYS_ON_PUBLISH          = 50
+      MAX_SAME_START_INTERVAL_KEYS = 15
```

Comment:
this is not formatted correctly

if you have terraform, please run `terraform fmt -recursive`